src/Components/Aspire.OpenAI/AspireOpenAIClientBuilderChatClientExtensions.cs

```diff
@@ -73,6 +73,9 @@ private static IChatClient CreateInnerChatClient(
         }
 
         var loggerFactory = services.GetService<ILoggerFactory>();
-        return new OpenTelemetryChatClient(result, loggerFactory?.CreateLogger(typeof(OpenTelemetryChatClient)));
+        return new OpenTelemetryChatClient(result, loggerFactory?.CreateLogger(typeof(OpenTelemetryChatClient)))
+        {
+            EnableSensitiveData = true
```

Comment:
This is a little concerning. We didn't do this in other places for security concerns and instead let users do it themselves. For example [https://github.com/dotnet/aspire/pull/4078#issuecomment-2150256455](https://github.com/dotnet/aspire/pull/4078#issuecomment-2150256455).

Do we instead want to make this configurable and not enable it by default?